### ISSUE TITLE
VxAdmin: election package auto-search only checks the root of the USB, would be helpful to check more

### ISSUE THREAD

**Issue by arsalansufi** [votingworks/vxsuite#5673]
The auto-search for election packages on VxAdmin often doesn’t find anything, even though it’s right on the USB, and I have to select it manually. In particular it's annoying that when I export an election package from VxAdmin, it doesn't later show up as an option in the menu because it's not at the root.

Issue automatically created from Slack feedback reported by drew: 
https://votingworks.slack.com/archives/CJU9MSC6S/p1730848259118389?thread_ts=1730848259.118389&cid=CJU9MSC6S

**Comment by adghayes** [comment]
Could perhaps close https://github.com/votingworks/vxsuite/issues/1572 if we do this.

**Comment by arsalansufi** [comment]
Agreed, went ahead and closed https://github.com/votingworks/vxsuite/issues/1572 as essentially done, with this issue as the main possible follow-up
